docs/source/get-involved/gsoc/projects_2025/brainglobe.md

```diff
@@ -45,9 +46,95 @@ Experience with Python, [NumPy](https://numpy.org/doc/stable/index.html) and/or
 
 **Further reading**
 <!-- The best pages include links to more detailed descriptions and related materials for each project. They might even include actual use cases! -->
+- [BrainGlobe developer guide](https://brainglobe.info/community/developers/index.html)
 - [cellfinder paper](https://doi.org/10.1371/journal.pcbi.1009074)
 - [cellfinder code](https://github.com/brainglobe/cellfinder)
 :::
 
+:::{dropdown} {fas}`brain;sd-text-primary` `cellfinder` support for two-dimensional brain images
+<!-- Description -->
+The [BrainGlobe `cellfinder` tool](https://brainglobe.info/documentation/cellfinder/index.html) is used to detect cells in large whole-brain images. It uses signal processing to find possible cell candidates and passes them to a customisable classifier to split the candidates into cells and no-cells. `cellfinder` relies heavily on `pytorch` and `keras`.
+
+`cellfinder` currently uses a residual neural network (ResNet) to classify cell candidates. This network is designed for three-dimensional whole-brain images, but most neuroscientists take images of two-dimensional slices of the brain. In this project, you would adapt `cellfinder` to support two-dimensional data. This would involve implementing a new neural network suitable for two-dimensional images, as well as refactoring some of the signal processing code.
```

Comment:
```suggestion
`cellfinder` currently uses a residual neural network (ResNet) to classify cell candidates. This network is designed for three-dimensional whole-brain images, but neuroscientists often take images of two-dimensional slices of the brain. In this project, you would adapt `cellfinder` to support two-dimensional data. This would involve implementing a new neural network suitable for two-dimensional images, as well as refactoring some of the image processing code.
```